### DISCUSSION TITLE
Is there a way to link the docs to the code for the corresponding components?

### DISCUSSION THREAD

**Discussion by pietrop** [vidstack/player#699] [Ideas]
_low priority idea 💡_

For some docs, where it applies, is there a way to link the docs to the code for the corresponding components?

eg [`PlayButton`](https://www.vidstack.io/docs/player/react/components/ui/play-button/) page could have a link to the github source. 

Rationale, is that I think these docs are great, but I also think often people use these library as black boxes, and having something like this (coupled with an architecture overview) might encourage checking out the source (for those who are interested in how it's implemented under the hood, those that are not can just ignore), an encourage more contributions, spotting of bugs etc..

_I am aware this blocked by the tooling refactor, but happy to help with PR when the time is right, if it's of interest._

**Comment by mihar-22** [comment]
Love the idea and not sure how we missed doing that 😅 As you said let's circle back as soon as we can.
